### ISSUE TITLE
Add Workflow Instructions for "GitHub Desktop"

### ISSUE THREAD

**Issue by Josh-McMillan** [Josh-McMillan/Community-Game#8]
Would it be possible to document the workflow instructions to help onboard people using GitHub Desktop?

While I have experience with commandline git (I've already had to use it in this repo alone to remove and untrack files, fixing gitignore issues), I and those inexperienced with git find it to be less confusing to use an all-in-one program, even if it is less powerful (which actually may benefit the team in this case.) Fundamentally, the workflow is nearly the same, the only difference is that common git commands are tied to buttons and menus, with the one exception being that staging files is done via a checkbox in the changed files list.

This can be an addendum to the current workflow instructions with a link at the top or an entirely different workflow page. Ideally, it would also include pictures in order to explain the UI in relation to our current workflow and teach our GitHub Desktop users how to properly create branches, commits, pulling changes from branches, and creating pull requests to other branches.

If you don't have/use GitHub Desktop, or are unfamiliar to the program, I would be glad to draft up a page/section for you to add.

**Comment by AlexanderKyron** [comment]
Sorry, I'm not particularly familiar with Github Desktop. Could you draft a section for it?

**Comment by Josh-McMillan** [comment]
Absolutely! I'll draft up an equivalent documentation page, and post it here 👍
